operations/helm/tests/metamonitoring-values-generated/mimir-distributed/templates/metamonitoring/recording-rules.yaml

```diff
@@ -0,0 +1,569 @@
+---
```

Comment:
after the suggested changes, you'll have to update https://github.com/grafana/mimir/blob/main/operations/helm/charts/mimir-distributed/ci/offline/metamonitoring-values.yaml to get the golden record back. Also we could turn on alerts as well.